fastly.toml

```diff
@@ -3,3 +3,18 @@ name = "Default starter for Rust"
 description = "Get to know the Fastly Compute@Edge environment with a basic starter kit that demonstrates routing, simple synthetic responses and overriding caching rules."
 authors = ["<oss@fastly.com>"]
 language = "rust"
+
+# The [setup] section is used by the Fastly CLI to provide a more integrated experience for shared 'Starter Kits'.
+# https://developer.fastly.com/reference/fastly-toml/
+#
```

Comment:
@cratelyn how does this sound to you?

Reply:
Lovely! Thank you so much :slightly_smiling_face: